### DISCUSSION TITLE
I want to overlay the UIScreen Sample Buffer on the camera capture Sample Buffer during the video broadcast

### DISCUSSION THREAD

**Discussion by cschoi724** [HaishinKit/HaishinKit.swift#1173] [Q&A]
This could have been captured alone using the IOUIScreenCaptureUnit, but it doesn't seem to work with VideoUnit.
I've temporarily modified the source a little bit so that I can overuse the IOUIScreenCaptureUnit on top of the VideoUnit, which is possible, but the code seems to be incomplete. Do you have any plans to officially support this?

And the background color of PixelBuffer created by IOUIScreen Capture Unit is black. Can I change it to a transparent color?

**Comment by shogo4405** [comment]
hmm. I would like to know specifically what kind of video you want to create? Depending on the purpose, it may be better not to use the IOUIScreenCaptureUnit.

**Comment by dhruv-sharma666** [comment]
> This could have been captured alone using the IOUIScreenCaptureUnit, but it doesn't seem to work with VideoUnit. I've temporarily modified the source a little bit so that I can overuse the IOUIScreenCaptureUnit on top of the VideoUnit, which is possible, but the code seems to be incomplete. Do you have any plans to officially support this?
> 
> And the background color of PixelBuffer created by IOUIScreen Capture Unit is black. Can I change it to a transparent color?

hi cschoi724 i am trying to overlay the UIScreen on capture sample for broadcasting  buffer, can you please provide the sample code to try it . I want to overlay a score board for my sports streaming . I am doing it for table tennis so the score will be changing frequently i would be reading the score through Socket.io . TIA
